src/utils/numbers.ts

```diff
@@ -7,20 +7,32 @@ type FormatCurrency = {
   currency?: string
   divisor?: number
   fractionDigits?: number
+  currencyPosition?: boolean
```

Comment:
why is this boolean? its confusing I just noticed

Reply:
You are right, i should make variants of it, "left" and "right"